crates/revm/revm-inspectors/src/tracing/builder/parity.rs

```diff
@@ -218,13 +231,116 @@ impl ParityTraceBuilder {
     pub fn into_transaction_traces(self) -> Vec<TransactionTrace> {
         self.into_transaction_traces_iter().collect()
     }
+
+    /// get addresses here because cant get them from parity trace otherwise
+    fn df_ordered_addresses(&self) -> Vec<Address> {
```

Comment:
what does `df` stand for here, not explained in docs.

please don't use abbreviations or make clear what this is standing for.

Reply:
sorry, depth first